### PR TITLE
Add bug-reports and dev-repo fields for most recent version of my packages; fix maintainer line for all my packages

### PR DESCRIPTION
**Pull request by darioteixeira**
Note: the fix of the maintainer line was motivated by the OPAM Weather Service reports.

### CODE REVIEW

File: packages/lambdoc/lambdoc.1.0-beta3/opam [hunk 1]
```diff
@@ -1,31 +1,31 @@
-opam-version: "1"
-maintainer: "dario.teixeira@nleyten.com"
+opam-version: "1.2"
+maintainer: "Dario Teixeira <dario.teixeira@nleyten.com>"
 authors: ["Dario Teixeira <dario.teixeira@nleyten.com>"]
-homepage: "http://lambdoc.forge.ocamlcore.org/"
+homepage: "https://lambdoc.forge.ocamlcore.org/"
```
Comment (dsheets) [inline packages/lambdoc/lambdoc.1.0-beta3/opam]: This URL gives me a TLS error because the cert is only for forge.ocamlcore.org and ocamlcore.org.

**Comment by dsheets** [comment]
`lambdoc.1.0-beta1` has a missing constraint on its `bookaml` dependency. It requires `"bookaml" {< "2.0"}` due to an interface change. Unfortunately, `bookaml.1.0` is the only version satisfying this constraint and it was removed in #3623. This makes `lambdoc.1.0-beta1` uninstallable.

**Comment by darioteixeira** [comment]
@dsheets: I've removed the lambdoc.1.0-beta1, given its uninstallable (and outdated) status.  I've also switched the homepage URLs from HTTPS to HTTP.  Finally, I upped the version number and added a few of the missing fields to placate opam-lint.

**Comment by darioteixeira** [comment]
Travis is complaining it cannot download OCamlnet.  Seems like a transient problem -- can someone restart it?

**Comment by dsheets** [comment]
With so many modified packages, Travis was timing out. Almost all of the changes were purely metadata and do not affect building so merging.

**Pull request merged**